index.js

```diff
@@ -4771,7 +4816,8 @@ export async function createCsharpBom(path, options) {
   }
   if (FETCH_LICENSE) {
     const retMap = await getNugetMetadata(pkgList, dependencies);
-    if (retMap.dependencies?.length) {
+    console.log(`dependencies re${retMap.dependencies?.length}`);
+    if (retMap.dependencies?.length > 1) {
```

Comment:
Why greater than 1?

Reply:
There is an empty set for this. It adds nothing.